### ISSUE TITLE
Add Configuration for RabbitMQ Port

### ISSUE THREAD

**Issue by luckyraul** [zulip/docker-zulip#377]
Where are settings below 
```
SETTING_MEMCACHED_LOCATION=host:port
SETTING_RABBITMQ_HOST=host
SETTING_REDIS_HOST=host
SETTING_REDIS_PORT=port
```

but there is no env for ```SETTING_RABBITMQ_PORT```

**Comment by alexmv** [comment]
This is because Zulip doesn't have this as a core setting.  I've opened a PR to add it; we'll get it backported into 6.x, and it should be in the next release of the Docker image.

**Comment by alexmv** [comment]
For reference, this was included in 6.2 and above.
